public/js/angular/controllers.js

```diff
@@ -377,7 +377,9 @@ angular
       $scope.reverse = false;
       $scope.predicate = 'created';
       $scope.hasPermissions = function (badge) {
-        return window.badgesPermissionsModel({
+        var badgesPermissionsModel = module.exports;
```

Comment:
I'm confused about what's being done here. Why not just return an object literal instead?